File: ObjectiveC/CollectionView101/CollectionView101/SampleData.m
//
//  SampleData.m
//  CollectionView101
//
//  Copyright © 2016 GrapeCity. All rights reserved.
//

#import "SampleData.h"

@implementation SampleData
- (id)initWithTitle:(NSString*)title subtitle:(NSString*)subtitle
{
    self = [super init];
    if(self)
    {
        self.title = title;
        self.subtitle = subtitle;
    }
    return self;
}
@end

File: Swift/CollectionView101/CollectionView101/SampleData.swift
//
//  SampleData.h
//  CollectionView101
//
//  Copyright © 2016 GrapeCity. All rights reserved.
//
import Foundation
class SampleData: NSObject {
    var title: String = ""
    var subtitle: String = ""
    var thumbnail: String = ""
    var link: String = ""
    var channelTitle: String = ""

    convenience init(title: String, subtitle: String) {
        self.init()
        self.title = title
        self.subtitle = subtitle
    }
}